packages/libs/api/sdk/README.md

```diff
@@ -111,6 +111,24 @@ client.nft
   .then((response) => console.log(response));
 ```
 
+### nft.getCollectionDetails
+
+| Argument | Values | Optional | Description             | Example                                    |
+| -------- | ------ | -------- | ----------------------- | ------------------------------------------ |
+| address  | string | ❌       | Contract address of NFT | 0x2106C00Ac7dA0A3430aE667879139E832307AeAa |
+
+```ts
+import { QuickNodeSDK } from '@quicknode/sdk';
+
+const client = new QuickNodeSDK();
+
+client.nft
+  .getCollectionDetails({
+    address: '0x2106C00Ac7dA0A3430aE667879139E832307AeAa',
```

Comment:
until we fix the address normalization, we should suggest that they use all lowercase?

Reply:
I think once we know more about how the casing works on the icy side, we can update our docs as needed

Reply:
makes sense 👍